eventsourcing/src/main/java/org/axonframework/eventsourcing/eventstore/jdbc/JdbcEventStorageEngine.java

```diff
@@ -764,6 +851,170 @@ public void setGapCleaningThreshold(int gapCleaningThreshold) {
         private int gapCleaningThreshold = DEFAULT_GAP_CLEANING_THRESHOLD;
```

Comment:
Nit: At the moment, the defaults of a `Builder` are not only defined on the `builder()` method, but also on class level of the `Builder` class.